### PR TITLE
[Azure Stack][Admin] Subscriptions for Azure Stack 1.5.0

### PR DESCRIPTION
**Pull request by deathly809**
## Description

Update to subscriptions.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1 [hunk 1]
```diff
@@ -121,19 +121,30 @@ Licensed under the MIT License. See License.txt in the project root for license
...
             # A URL to the main website for this project.
-            # ProjectUri = ''
+            ProjectUri   = 'https://github.com/Azure/azure-powershell'
 
             # A URL to an icon representing this module.
             # IconUri = ''
 
             # ReleaseNotes of this module
-            # ReleaseNotes = ''
+            ReleaseNotes = '## 2018.8.12 - Version 0.2.0
```
Comment (deathly809) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1]: Update the date.

Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1]: Will update all dates when we do PR to update AzureStack rollup module.

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1 [hunk 2]
```diff
@@ -121,19 +121,30 @@ Licensed under the MIT License. See License.txt in the project root for license
...
+            * Module dependencies updated
+                * AzureRM.Profile >= 5.3.4
+                * AzureRM.Resources >= 6.0.2
+            * Support handling names of nested resources
+                * TODO
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1]: None, removed.

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProvider.ps1 [hunk 3]
```diff
@@ -32,7 +32,7 @@ function Get-AzsDelegatedProvider
     param(
         [Parameter(Mandatory = $true, ParameterSetName = 'Get', Position = 0)]
         [ValidateNotNullOrEmpty()]
-        [System.String]
+        [System.Guid]
```
[the remark this answers is not in the document]
Comment (bganapa) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProvider.ps1]: Technically SubscriptionId can be string. ARM allows it, we should just keep it as string

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1 [hunk 4]
```diff
@@ -121,19 +121,27 @@ Licensed under the MIT License. See License.txt in the project root for license
...
+            ProjectUri   = 'https://github.com/Azure/azure-powershell'
 
             # A URL to an icon representing this module.
             # IconUri = ''
 
             # ReleaseNotes of this module
-            # ReleaseNotes = ''
+            ReleaseNotes = '## 2018.8.12 - Version 0.2.0
+            * Module dependencies updated
+                * AzureRM.Profile >= 5.3.4
```
Comment (bganapa) [inline src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Azs.Subscriptions.Admin.psd1]: lets remove the versions from the module release notes? we will keep it on the uber module..

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProvider.ps1 (hunk 3 above)
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProvider.ps1]: @bganapa  You are the one who said to change it :P

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsPlanToOffer.ps1 [hunk 5]
```diff
@@ -77,61 +77,53 @@ function Add-AzsPlanToOffer {
...
-            $dontCheck = $true
-            try {
-                $_offer = Get-AzsManagedOffer -Name $OfferName -ResourceGroupName $ResourceGroupName
-                $_planId = $_offer.BasePlanIds | Where-Object { $_ -like "*$PlanName" }
-                $_plan = Get-AzsPlan -ResourceId $_planId
-                $dontCheck = $_plan -eq $null
-            } catch {
-                # No-op
+            $_offer = Get-AzsManagedOffer -Name $OfferName -ResourceGroupName $ResourceGroupName
+            $_planId = $_offer.BasePlanIds | Where-Object { $_ -like "*$PlanName" }
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsPlanToOffer.ps1]: Not sure, I didn't write this part of the code.  I will add it if you feel like we should have it.

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProvider.ps1 (hunk 3 above)
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDelegatedProvider.ps1]: There are 9 other cmdlets which take in a GUID which we internally convert to strings.  I think we either change them all or leave this unchanged.

File: src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsPlanToOffer.ps1 (hunk 5 above)
[the remark this answers is not in the document]
Comment (deathly809) [reply src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsPlanToOffer.ps1]: We only support / characters in Ids.  I have added it.

**Pull request merged**